examples/export-map.js

```diff
@@ -16,6 +22,18 @@ const map = new Map({
       }),
       opacity: 0.5,
     }),
+    new HeatmapLayer({
+      source: new VectorSource({
+        features: [
+          new Feature(new Point([0, 1000])),
+          new Feature(new Point([-1000, 0])),
+          new Feature(new Point([1000, 0])),
+          new Feature(new Point([0, -1000])),
+        ],
+      }),
+      opacity: 0.5,
+      preserveDrawingBuffer: true,
+    }),
```

Comment:
We should avoid mixing tests and examples.  This change is making the example feel more like a test (potentially confusing to users looking to learn how to export a map).